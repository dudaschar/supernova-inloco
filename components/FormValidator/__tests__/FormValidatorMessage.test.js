import { shallow } from 'enzyme'
import React from 'react'

import FormValidatorMessage from '../Message'

const buildWrapper = props =>
  shallow(
    <FormValidatorMessage
      className="test-class"
      message="Custom message"
      {...props}
    />
  )

describe('FormValidatorMessage', () => {
  it('should render the message', () => {
    expect(buildWrapper()).toMatchSnapshot()
  })
})
