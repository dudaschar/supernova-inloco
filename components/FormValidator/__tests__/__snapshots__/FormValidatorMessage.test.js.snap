// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormValidatorMessage should render the message 1`] = `
<div
  className="inloco-form-validator__field test-class"
>
  Custom message
</div>
`;
